File: javascript/DaySeven/TaskNine.js
let check = 1;
let counter = 1;
let totalSum = 0;
let square = 1



for (check = 1; check <= 10; check++){
	if (check % 4 == 0){
		
		let multiples = 1;
		let sum = 0;
		for (counter = 1; counter <= 5; counter++){		
			multiples = multiples * check;
			sum = sum + multiples;
	
}
totalSum += sum;
square = totalSum * totalSum;

	
		}
}
console.log(square);
	


		

File: python/DaySeven/task_nine.py
check = 1
counter = 1
total_sum = 0



for check in range(1,11):
	if (check % 4 == 0):
		
		multiples = 1
		sum = 0
		for counter in range(5):		
			multiples = multiples * check
			sum += multiples
		total_sum += sum
		square  = total_sum * total_sum

print(square, end= " ")	
		

		